### PR TITLE
Terraform 0.14.7 Update

### PR DESCRIPTION
**Pull request by onlydole**
* Update Docker depencencies
* Update Terraform version
* Add GitHub workflows for linting

### CODE REVIEW

**Review by tracypholmes — APPROVED** [review]
LGTM 👍🏾

File: .github/workflows/terraform.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    name: Terraform
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v1.2.1
```
Comment (ksatirli) [inline .github/workflows/terraform.yaml]: Thinking you could change this to `v1` (from `v1.2.1`) to bring it in line with all other version references.

File: provider.tf [hunk 2]
```diff
@@ -1,6 +1,5 @@
 provider "docker" {
-  version = "~> 2.7"
```
Comment (ksatirli) [inline provider.tf:2]: good change!

**Review by ksatirli — APPROVED** [review]
One small nit that could be fixed, otherwise 👍🏼

File: .github/workflows/terraform.yaml (hunk 1 above)
Comment (onlydole) [reply .github/workflows/terraform.yaml]: Absolutely 👍🏼

**Review by ksatirli — APPROVED** [review]

**Pull request merged**